docs/community/100-days-of-oss.md

```diff
@@ -22,8 +22,15 @@ There are numerous ways to participate in the #100DaysOfOSS challenge, including
 - create or participate in discussions related to OSS topics;
 - write a blog post or create content that supports an OSS project; 
 - update or write documentation to improve clarity and usability
-- create content: give a talk or presentation on OSS, participate in or even start a Twitter Space write a blog post or create a video;  
-- maintain a project: review pull requests, triage issues, and respond to questions;
+- Content creation: Love creating content? Here's how you can use this skill for the challenge:
+  - Give a talk or presentation on OSS.
+  - Participate in or even start a Twitter Space.
+  - Write a blog post or create a video.
+
+- Project management: Want to improve your project management skills? Need to get better at communicating or giving feedback to others? Here's how you can use this skill for the challenge:
+  - Review pull requests.
+  - Triage issues.
+  - Respond to questions.
 - support contributors working on OSS projects, providing guidance and support.
 
 To keep track of your progress, post on social media, your blog, or any other platform you prefer with what progress you made, the day of the challenge indicated by 'D'and include the hashtag #100DaysOfOSS. For example, if you're on day one, you could say, "Today, I reviewed the documentation for the [OpenSauced/Insights](https://github.com/open-sauced/insights) repository. D1 #100daysOfOSS." Then, on day two, you would continue with D2, and so on.
```

Comment:
[nitpick/proofreading] This part is off to a great start! :) Good idea to provide an example of how participants can post their progress. I think it can be improved by mentioning the word "contributions" and separating the example to the bottom as shown below: 
```suggestion
To keep track of your progress, post about your contributions on social media, your blog, or any other platform you prefer. Also, add the day of the challenge indicated by the initial 'D' and include the hashtag `#100DaysOfOSS`. For example, if you're on day one, you could do the following:
"Today, I reviewed the documentation for the [OpenSauced/Insights](https://github.com/open-sauced/insights) repository. D1 #100daysOfOSS." Then, you would do the same for the second day, and so on.
```